README.md

```diff
@@ -18,11 +18,21 @@ Table sections are objects that conform to the `Section` protocol, most of the t
 
 # Installation
 
-Setting up your app to use Thunder Table is a simple.
+Setting up your app to use ThunderTable is a simple and quick process. You can choose between a manual installation, or use Carthage.
 
-+ Drag ThunderTable.xcodeproj
-+ Add ThunderTable.framework to your Embedded Binaries.
-+ Wherever you want to use ThunderTable use `import ThunderTable`
+## Carthage
+
+- Add `github "3sidedcube/ThunderTable" == 1.5.0` to your Cartfile.
```

Comment:
If you're changing the Cartfile here should prob change the version to the (upcoming?) latest! :)